website/docs/blueprints/ai-ml/jupyterhub.md

```diff
@@ -168,7 +168,7 @@ kubectl port-forward svc/proxy-public 8080:80 -n jupyterhub
 
 Select server options: Upon sign-in, you’ll be presented with a variety of Notebook instance profiles to choose from. For this time-slicing feature demonstration, we’ll be using the **Data Science (GPU + Time-Slicing – G5)** profile. Go ahead and select this option and choose the Start button.
 
-![alt text](img/image-1.png)
+![alt text](img/notebook-server-list.png)
```

Comment:
Let's add one paragraph about [Elyra](https://github.com/elyra-ai/elyra) and its benefits.